app/src/main/java/us/kikin/apps/android/bgplayer/ui/show/ShowListFragment.kt

```diff
@@ -48,6 +46,11 @@ class ShowListFragment : Fragment(), VideoItemClickListener, ShowItemClickListen
         )
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        _binding = null
```

Comment:
nit: set it to null before the super call, so it mirrors what you do in onCreateView

Reply:
I would generally agree. But this is how it's written in the official docs.. Maybe there's a reason?
https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
It's probably a detail they thought was irrelevant, there is no reasons not to do it before `super.onDestroy()`. The super won't access `_binding` because it doesn't know about it, so I am confident it's safe.